editors/code/src/lang_client.ts

```diff
@@ -0,0 +1,16 @@
+import * as lc from "vscode-languageclient/node";
+import * as vscode from "vscode";
+
+export class RaLanguageClient extends lc.LanguageClient {
+    override error(message: string, data?: any, showNotification?: boolean | "force"): void {
```

Comment:
Do you know what data might contain? Does it contain the error code by any chance? Would be nice if we could match on that instead, because then we can filter out some specific errors that VSCode raises (on non panicking requests) unconditionally (and log them instead).

Reply:
It seems like this is the `ResponseError` emitted by r-a, so it does contain the error code. I changed this to override `handleRequestFailed` instead of `error`, and set `showNotification` to `false` if we detect that it's an `InternalError`.